### DISCUSSION TITLE
AnvilLoader giving me an empty instance

### DISCUSSION THREAD

**Discussion by lunasoldev** [Minestom/Minestom#1838] [Help] [answered]
![image](https://user-images.githubusercontent.com/66495367/234898609-6a07e005-a4f0-4f56-8738-c91998e936e4.png)
I am using the message above, but the instance seems to be empty, I only fall down even though it should have a stone platform at the spawn location.

**Comment by ZakShearman** [comment]
Firstly I would recommend avoiding static use all over the place.

You should set the chunk loader when you create the instance since you should have the ability to do so, not when the player logs in. Can you also show your code for creating the instance as well as the folder structure for when your JAR is and where the world is.

**Comment by Tofaa2** [comment]
Do you mind showing us the "worlds/world" folder and its contents?

**Reply by lunasoldev** [reply, marked as answer]
I managed to fix it, it had nothing to do with the world itself, i just had to use PlayerEvents.class.getClassLoader().getResource("worlds/world").getPath(); I was getting it from the resources folder.
